### PR TITLE
Solved: Add rosetta (interpeter) #300

### PR DESCRIPTION
**Pull request by adriantintpilver**
Create rosetta-2 pldb

### CODE REVIEW

File: database/things/rosetta-2.pldb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+title Rosetta-2
+appeared 2006
+type pl
+creators Apple Inc
+country United State
+
+wikipedia https://en.wikipedia.org/wiki/Rosetta_(software)
+ related dynamic binary translator
```
Comment (breck7) [inline database/things/rosetta-2.pldb:8]: this line is causing the test failure.

**Review by breck7 — CHANGES_REQUESTED** [review]
looks good! just please fix the failing test (probably just remove that "related" line. the related line must contain only permalinks to other PLDB entries)

**Comment by adriantintpilver** [comment]
Thanks Breck, has can't found the line with error and It was driving me crazy :-D
In addition add a "What Is Rosetta 2 And How to Install It on M1 Mac" link in documentation.

**Comment by breck7** [comment]
Great work @adriantintpilver !

**Pull request merged**